src/site/content/en/learn/html/semantic-html/index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 'Semantic HTML'
+authors:
+  - estellweyl
+description: Using the correct HTML elements to describe your document content.
+date: 2022-09-08
+tags:
+  - html
+---
+
+With over 100 HTML elements, and the ability to create [custom elements](), there are infinite ways to mark up your content; but some ways—notably _semantically_—are better than others. 
+
+_Semantic_ means "relating to meaning". Writing semantic HTML means using HTML elements to structure your content based on each element's meaning, not its appearance. 
+
+You haven't covered many HTML elements yet, but even without knowing HTML, the following two code snippets show how semantic markup can give content context. Both use a word count instead of _ipsum lorem_ to save some scrolling—use your imagination to expand "thirty words" into 30 words:
```

Comment:
```suggestion
We haven't covered many HTML elements yet, but even without knowing HTML, the following two code snippets show how semantic markup can give content context. Both use a word count instead of _ipsum lorem_ to save some scrolling—use your imagination to expand "thirty words" into 30 words:
```

Reply:
We prefer second person (you, not we) so I prefer the original https://web.dev/handbook/grammar/

Reply:
I knew that when i made the change, but I felt the "you" here was really accusatory. So maybe "This series hasn't covered...."